src/Dropdown/Dropdown.tsx

```diff
@@ -322,11 +322,11 @@ const Dropdown: React.FunctionComponent<DropdownProps> = (props: DropdownProps):
     // Display the custom dropdown with native elements if prop is set to native
     if (props.native) {
         return (
-            <>
+            <div className="custom-native-dropdown">
```

Comment:
Is this issue only found in the **native** variant?

Reply:
No, both variants previously used the `alert alert-danger` class names only.